trento/xml/article_sap_trento.xml

```diff
@@ -231,6 +233,19 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
                </para>
             </listitem>
           </itemizedlist>
+           <formalpara>
+            <title>systemd deployment:</title>
+            <para> The deployment using RPM packages can be done manually step-by-step
+             in a traditional fashion or it can be automated using an ansible playbook. 
```

Comment:
I think ansible should be written in uppercase:

```suggestion
             in a traditional fashion or it can be automated using an Ansible playbook. 
```